include/path_tracking_pid/details/second_order_lowpass.hpp

```diff
@@ -4,26 +4,41 @@
 
 namespace path_tracking_pid::details
 {
-// Error tracker for the last 3 error and filtered error values.
+/**
+ * @brief Discrete time second order lowpass filter
+ */
 class SecondOrderLowpass
 {
 public:
-  // Pushes the given value to the errors FIFO buffer. A corresponding filtered error value is calculated and pushed
-  // to the filtered errors FIFO buffer.
-  void push(double value);
+  /**
+   * @brief Construct a SecondOrderLowpass instance with NaNs
+   */
+  SecondOrderLowpass();
```

Comment:
If you change the default member initializer of cutoff and damping, you could default this contructor.

Reply:
I've done that refactor